portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,10 +26,21 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  private static final ImmutableList<String> FRUIT_FACTS = ImmutableList.of("A strawberry is not an actual berry", 
+  "Kiwis contain more Vitamin C than oranges", "The pineapple is actually a berry");
```

Comment:
nit: let's format this a bit for readability.

..... FRUIT_FACTS = 
  ImmutableList.of(
    " Strawberry...",
    "Kiwi",
    "Oranges");

Not sure if the formatting will show up but "ImmutableList.of()" is indented two spaces in from the declaration of "private", and the strings are two in from that. I'm not sure if your editor/shell has auto-formatting tools/commands but those would automatically fix them.

Reply:
Done! Thanks for pointing this out